src/pmdas/resctrl/sys-fs-resctrl.mount

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/sys-fs-resctrl.mount
```

Comment:
@scox I'd drop this comment too since its possibly incorrect already, better to not have the comment in that case IMO (rather than updating it I mean)